library/dd-sdk-android-webview/src/test/kotlin/com/datadog/android/webview/internal/MixedWebViewEventConsumerTest.kt

```diff
@@ -94,7 +94,9 @@ internal class MixedWebViewEventConsumerTest {
         // Then
         argumentCaptor<Pair<JsonObject, String>> {
             verify(mockLogsEventConsumer).consume(capture())
-            assertThat(firstValue.first).isEqualTo(fakeBundledEvent)
+            // toString call because of how Gson is comparing Float/Double vs LazilyParsedNumber
+            // for JsonPrimitive https://github.com/google/gson/issues/1864
```

Comment:
I think there is a solution for that:
https://github.com/DataDog/dd-sdk-android/blob/144d7babd3928157bf417f08a01cf85042c8f7a3/instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integration/sessionreplay/SrTest.kt#L44

Reply:
it can be used, but I would rather just compare strings here, given that anyway during the WebView ingerstion flow we don't work with Java/Kotlin types.